exercises/chapter6/test/Solutions.purs

```diff
@@ -0,0 +1,49 @@
+module Test.Solutions where
+
+import Prelude
+
+newtype Complex
+  = Complex
+  { real :: Number
+  , imaginary :: Number
+  }
```

Comment:
This file contains code snippets directly from the chapter text for users to complete.